chanbackup/pubsub.go

```diff
@@ -278,9 +279,11 @@ func (s *SubSwapper) backupUpdater() {
 			)
 			for i, closedChan := range chanUpdate.ClosedChans {
 				log.Debugf("Removing channel %v from backup "+
-					"state", newLogClosure(func() string {
-					return chanUpdate.ClosedChans[i].String()
-				}))
+					"state", lnutils.NewLogClosure(
+					func() string {
+						return chanUpdate.
+							ClosedChans[i].String()
```

Comment:
Can't you just do `chanUpdate.ClosedChans[i].String`?

Reply:
even better, changed